lms/djangoapps/teams/views.py

```diff
@@ -620,22 +623,37 @@ def get(self, request):
                 'user_message': _(u"The ordering {ordering} is not supported").format(ordering=ordering),
             }, status=status.HTTP_400_BAD_REQUEST)
 
-        page = self.paginate_queryset(topics)
-        serializer = self.pagination_serializer_class(page, context={'course_id': course_id, 'sort_order': ordering})
+        # Always sort alphabetically, as it will be used as secondary sort
+        # in the case of "team_count".
+        topics = get_alphabetical_topics(course_module)
+        if ordering == 'team_count':
```

Comment:
There doesn't seem to be much precedent for this, but I wonder if it makes sense (as this method continues to grow) to include some private methods on `TopicListView` that can help make the `get` method more readable (It's over 60 lines long with this addition).  Not a deal breaker, but it may be worth having the discussion at least.